package.json

```diff
@@ -7,7 +7,19 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "dev": "webpack-dev-server --mode development",
     "build": "webpack --mode production",
-    "start": "npm run build && serve -s dist"
+    "start": "npm run build && serve -s dist",
+    "format": "eslint --fix src/**/*.js"
```

Comment:
I tried to run eslint on my computer and got this kind of error
<img width="1165" alt="Screenshot 2023-04-18 at 18 04 35" src="https://user-images.githubusercontent.com/49757277/232837188-9035f031-9995-484e-a7ff-1b610e2ce820.png">